easybuild/easyconfigs/m/Mercurial/Mercurial-5.7.1-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = "PythonPackage"
+
+name = 'Mercurial'
+version = '5.7.1'
+
+homepage = 'http://mercurial.selenic.com/'
+description = """Mercurial is a free, distributed source control management tool. It efficiently handles projects
+of any size and offers an easy and intuitive interface.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://www.mercurial-scm.org/release/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['cb5139144ccb2ef648f36963c8606d47dea1cb0e22aa2c055d6f860ce3fde7b0']
+
+dependencies = [
+    ('Python', '3.8.6')
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/hg'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/mercurial'],
+}
```

Comment:
The default sanity check for a `PythonPackage` will already check for a non-empty `site-packages`. This means that we can remove the whole of this section and replace it with
```
sanity_check_commands = ["hg --help"]
```